src/media/js/addon/components/addonSubnav.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {ReverseLink} from 'react-router-reverse';
+
+import {Subnav} from '../../site/components/subnav';
+
+
+export default class AddonSubnav extends React.Component {
+  render() {
+    return (
+      <Subnav>
+        <ReverseLink to="addon-dashboard">My Add-ons</ReverseLink>
+        <ReverseLink to="addon-submit">Submit an Add-on</ReverseLink>
```

Comment:
never mind about my previous deleted comment. maybe these should be alphabetical?